test/e2e/tests/user_guide_tests/test_block_unblock_user.py

```diff
@@ -0,0 +1,102 @@
+import allure
+import pytest
+from allure_commons._allure import step
+
+import driver
+from constants.community_settings import BlockPopupWarnings, ToastMessages
+from gui.main_window import MainWindow
+from . import marks
+import configs
+import constants
+from constants import UserAccount
+
+pytestmark = marks
+
+
+@allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/738772',
+                 "Block or unblock someone in Status")
+@pytest.mark.case(738772)
```

Comment:
```suggestion
# @pytest.mark.case(738772)
```
for now, so we wont fail all the nightly runs. I did not change the mapping to test rail yet